portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,6 +19,15 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
```

Comment:
There should only be one blank line between static and non static imports. See https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

As Aleksey said in a separate comment, it's easiest to fix this with a formatter. You can try:
https://github.com/google/google-java-format